assets/collage.css

```diff
@@ -5,6 +5,8 @@
 
 .collage {
   display: grid;
+  position: relative;
+  z-index: 0;
```

Comment:
TL;DR - the shadows for cards have a `z-index` of `-1` and when a gradient is applied to the section, the shadows disappear.  They end up being put behind the background - not 100% sure why, but it might be that it's treated as an "image" technically.

I'll look into the modal issue.  Worst case, we might need to remove the setting from this section.